components/schemes/Container/src/MDCContainerScheme.h

```diff
@@ -49,8 +49,8 @@
 __attribute__((objc_subclassing_restricted)) @interface MDCContainerScheme
```

Comment:
Please be sure to update the header documentation now that properties are no longer `nil`.  The defaults should be explicitly documented.

Reply:
Done

Reply:
This is still not done - please fix the header documentation above.

Reply:
Good catch, I didn't see that. It has been fixed.